src/main/java/io/github/furstenheim/CopyNode.java

```diff
@@ -81,6 +81,10 @@ FlankingWhiteSpaces flankingWhitespace () {
             } else {
                 textContent = element.outerHtml();
             }
+            // Don't put extra spaces for a line break
+            if (textContent.equals("\n")) {
+                return new FlankingWhiteSpaces("", "");
```

Comment:
Sorry, could you provide an example of what we were getting before and what we are getting now with the new version?
Thanks

Reply:
This test was failing after updating:
```json
{
    "name": "br",
    "options": null,
    "input": "More<br>after the break",
    "output": "More  \nafter the break"
}
```

with this exception:
```
Expected: "More  \nafter the break"
     but: was "More   \n after the break"
```